### PR TITLE
Clarify cloud project import

### PR DESCRIPTION
**Pull request by yomovh**
Clarify documentation based on issue https://github.com/ovh/terraform-provider-ovh/issues/342

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-check#1283.0 ✔ 
* 
  * checks ✔

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-check#1282.0 ✔ 
* 
  * checks ✔

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-testacc#1283.0 ✘  
* 
  * checks ✘

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-testacc#1282.0 ✘  
* 
  * checks ✘

File: website/docs/r/cloud_project.html.markdown [hunk 1]
```diff
@@ -84,3 +84,9 @@ Id is set to the order Id. In addition, the following attributes are exported:
 * `project_name` - openstack project name
 * `project_id` - openstack project id
 * `status` - project status
+
+## Import
+Cloud project can be imported using the `order_id` that can be retrieved in the [order page](https://www.ovh.com/manager/#/dedicated/billing/orders/orders) at the creation time of the public cloud project 
+```bash
+$ terraform import ovh_cloud_project_database.my_cloud_project order_id
```
Comment (scraly) [inline website/docs/r/cloud_project.html.markdown:91]:
```suggestion
$ terraform import ovh_cloud_project.my_cloud_project order_id
```

File: website/docs/r/cloud_project.html.markdown [hunk 2]
```diff
@@ -84,3 +84,9 @@ Id is set to the order Id. In addition, the following attributes are exported:
 * `project_name` - openstack project name
 * `project_id` - openstack project id
 * `status` - project status
+
+## Import
+Cloud project can be imported using the `order_id` that can be retrieved in the [order page](https://www.ovh.com/manager/#/dedicated/billing/orders/orders) at the creation time of the public cloud project
```
Comment (scraly) [inline website/docs/r/cloud_project.html.markdown:89]:
```suggestion
Cloud project can be imported using the `order_id` that can be retrieved in the [order page](https://www.ovh.com/manager/#/dedicated/billing/orders/orders) at the creation time of the Public Cloud project. 
```

**Review by scraly — CHANGES_REQUESTED** [review]
Thanks for the PR :-) 

I added two fixes/suggestions.

Moreover, I see the list of attributes are not up to date, is it possible to update the list?

Example of actual attributes:

```
resource "ovh_cloud_project" "my_cloud_project" {
    access       = "full"
    description  = "MY_PROJECT"
    id           = "123456789"
    order        = [
        {
            date            = "2021-09-20T15:14:37+02:00"
            details         = [
                {
                    description     = "Activation d'un projet /cloud"
                    domain          = "xxxx"
                    order_detail_id = 123456789
                    quantity        = "1"
                },
                {
                    description     = "Public Cloud Project cloud project prices jusqu'au 01/10/2021"
                    domain          = "xxxxx"
                    order_detail_id = 123456789
                    quantity        = "1"
                },
            ]
            expiration_date = "2021-10-04T23:29:59+02:00"
            order_id        = 123456789
        },
    ]
    project_id   = "xxxxx"
    project_name = "xxx"
    status       = "ok"
}
```

And another question, I see in the doc a warning: "This resource is in beta state. Use with caution."

Can you see if this warning block can be remove now? :) 

Thanks

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-check#1288.0 ✔ 
* 
  * checks ✔

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-check#1287.0 ✔ 
* 
  * checks ✔

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-testacc#1288.0 ✘  
* 
  * checks ✘

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-testacc#1287.0 ✘  
* 
  * checks ✘

**Comment by yomovh** [comment]
Attributes added.

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-check#1290.0 ✔ 
* 
  * checks ✔

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-check#1289.0 ✔ 
* 
  * checks ✔

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-testacc#1290.0 ✘  
* 
  * checks ✘

**Comment by ovh-cds** [comment]
CDS Report terraform-provider-ovh-testacc#1289.0 ✘  
* 
  * checks ✘

**Comment by scraly** [comment]
Thanks :)

**Pull request merged**